http_server.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+import socket
+import email.utils
+import mimetypes
+
+
+def http_server():
+    server_socket = socket.socket(
+        socket.AF_INET,
+        socket.SOCK_STREAM,
+        socket.IPPROTO_IP)
+    server_socket.bind(('127.0.0.1', 50000))
+    server_socket.listen(10)
+    try:
+        while True:
+            conn, addr = server_socket.accept()
+            buffsize = 16
+            msg = ""
+            done = False
+            while not done:
+                part = conn.recv(buffsize)
+                if len(part) < buffsize:
+                    done = True
+                msg = b"{}{}".format(msg, part)
+            out = parse_request(msg)
+            conn.sendall(out)
+            conn.close()
+    except KeyboardInterrupt:
+        server_socket.close()
+
+
+def response_ok(uri):
+    date = email.utils.formatdate(usegmt=True)
```

Comment:
In this stage of the assignment, `response_ok` is not supposed to take *any* arguments.